build/Program.fs

```diff
@@ -54,103 +52,44 @@ let publish() =
         then failwith "Publish failed"
 
 let buildCraftSchema() =
-    if Shell.Exec(Tools.dotnet, "run -p Snowflaqe.fsproj -- --config ../samples/craft-cms/snowflaqe.json --generate", path [ solutionRoot; "src" ]) <> 0 then
+    if Shell.Exec(Tools.dotnet, "run -f netcoreapp3.1 -- --config ../samples/craft-cms/snowflaqe.json --generate", path [ solutionRoot; "src" ]) <> 0 then
```

Comment:
Please don't use a specific target framework here. Why do you need this?

Reply:
Because we have both `netstandard2.0` and `netcoreapp3.1` targets. And `dotnet` requires to specify which one we want to execute

Reply:
Maybe will we extract that to a constant?

Reply:
OK, I see. A constant sounds good